src/palace/manager/feed/serializer/opds.py

```diff
@@ -314,12 +317,17 @@ def _serialize_feed_entry(
                 if attrib == "text":
                     entry.text = value
                 else:
+                    attribute_mapping = self._get_attribute_mapping()
                     entry.set(
-                        ATTRIBUTE_MAPPING.get(attrib, attrib),
+                        attribute_mapping.get(attrib, attrib),
                         value if value is not None else "",
                     )
         return entry
 
+    @abc.abstractmethod
+    def _get_attribute_mapping(self) -> dict[str, str]:
```

Comment:
Minor: Since subclasses are expected to implement these, its would be nice to have a doc comment with a small bit of explanation of what an implementation should provide for this function.